debian/postrm

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+STAMP_FILE=/var/cache/ubuntu-advantage-tools/ubuntu-advantage-status.stamp
```

Comment:
I'd s/STAMP/CACHE/ everywhere (including filename suffix), since it's not just an empty stamp file, it actually contains data

Reply:
Funy, I copied this syntax/idea from other similar scripts, I guess "stamp" got abused indeed